src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -189,8 +202,8 @@ public boolean equals(Object other) {
             EditModuleDescriptor e = (EditModuleDescriptor) other;
 
             return getModuleCode().equals(e.getModuleCode())
-                    && getModularCredits().equals(e.getModularCredits())
-                    && getTags().equals(e.getTags());
+                && getModularCredits().equals(e.getModularCredits())
```

Comment:
Should we add a state check for module title equality too?

Reply:
done!